sw.js

```diff
@@ -0,0 +1,7 @@
+
+// Service Workers are replacing Background in Chrome MV3
+const bra = (greeting) => {
```

Comment:
Could potentially be simplified as

```javascript
const bra = greeting => `${greeting}, bra!`
```

as single-argument lambdas don't need parentheses and single-statement lambdas don't need explicit blocks.

Reply:
It works as I intended but not as service-workers are intended at the moment. For now I'm just testing that the files can communicate with each other as expected. Full service-worker build and test is coming up.